### ISSUE TITLE
Not letting me select puzzle from 2020-10-13

### ISSUE THREAD

**Issue by samtell21** [apexatoll/cliptic#2]
Supposedly, this is the earliest available puzzle, but when I try to select it, I get an exit code 1 with the following message:

```
Invalid date passed 2020-10-13
Earliest date 2020-10-13
Thanks for playing!
```
Looks like a little bug...  Otherwise, working great!

EDIT: Thought I'd mention that 2020-10-14 and beyond are just fine.

**Comment by apexatoll** [comment]
Thanks for letting me know - probably a missing equals sign somewhere!

**Comment by apexatoll** [comment]
There are a couple of bugs that will be fixed in the upcoming version in the next couple of days

**Comment by apexatoll** [comment]
This bug is now fixed in 0.1.2
